p2p/src/network.rs

```diff
@@ -59,6 +59,8 @@ where
     pub new_peers: HashMap<ConnectionId, Addr<Peer<T, K, E>>>,
     /// Current [`Peer`]s in [`Peer::Ready`] state.
     pub peers: HashMap<PublicKey, RefPeer<T, K, E>>,
+    /// Connecting to peer id's.
```

Comment:
```suggestion
    /// The ids of peers to connect to
```
```suggestion
    /// The ids of connecting peers
```
also: why so we say `id` if it's a vector of public keys?
```suggestion
    /// The public keys of connecting peers
```